ThunderCloud/GridPage.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  GridPage.swift
+//  ThunderCloud
+//
+//  Created by Simon Mitchell on 16/02/2018.
+//  Copyright © 2018 threesidedcube. All rights reserved.
+//
+
+import ThunderCollection
+import ThunderTable
+
+/// A subclass of `TSCCollectionViewController` for displaying a CMS grid page
+open class GridPage: CollectionViewController, StormObjectProtocol {
+	
+	/// An array of dictionaries which contain custom attributes fot the `StormObject`
+	public var attributes: [[AnyHashable : Any]]?
+	
+	private let dictionary: [AnyHashable : Any]
```

Comment:
Could you document this? It's unusual that we store a dictionary for a storm object so would be good to write up why we do.